openshift/e2e-tests.sh

```diff
@@ -18,7 +18,7 @@ failed=0
 
 (( !failed )) && install_serverless || failed=1
 
-(( !failed )) && install_knative_eventing || failed=1
+##(( !failed )) && install_knative_eventing || failed=1
```

Comment:
any reason we're commenting this out vs just removing?

Reply:
ah, to test it ... :-)